addons/survey/views/survey_templates_statistics.xml

```diff
@@ -217,13 +220,20 @@
                                 </t>
                             </td>
                             <td>
-                                <t t-if="question.question_type == 'text_box'">
-                                    <span t-if="no_print_url" t-field="input_line.value_text_box" />
-                                    <a t-else="" t-att-href="input_line.user_input_id.get_print_url()" t-field="input_line.value_text_box"></a>
+                                <!-- If answer already covered by filter or if there is only one line to display, do not allow filtering on it again -->
+                                <t t-if="question.id in disabledQuestionsFilters or len(table_data) == 1">
+                                    <span t-field="input_line.value_text_box" t-if="question.question_type == 'text_box'"/>
+                                    <span t-field="input_line.value_char_box" t-if="question.question_type == 'char_box'"/>
```

Comment:
Really minor comment: best to use alphabetical order as fallback if there is no other order that makes sense.
```suggestion
                                    <span t-field="input_line.value_char_box" t-if="question.question_type == 'char_box'"/>
                                    <span t-field="input_line.value_text_box" t-if="question.question_type == 'text_box'"/>
```